go-controller/cmd/ovnkube/ovnkube.go

```diff
@@ -164,6 +170,11 @@ func runOvnKube(ctx *cli.Context) error {
 		ovnController := ovn.NewOvnController(clientset, factory)
 		ovnController.NodePortEnable = nodePortEnable
 		ovnController.Run()
+		err := clusterController.RebuildClusterNodes(master, ovnController)
```

Comment:
I think RebuildClusterNodes should be called before calling Run(). This is because as soon as Run() is called, you will start getting events.

Reply:
Done